timers: add getActiveTimers that returns all active timers (#28769)
nodejs/node: doc/api/timers.md

Cancels a `Timeout` object created by [`setTimeout()`][].

## Diagnostics

### getActiveTimers()
<!-- YAML

> added: REPLACE ME

Reviewer: ```suggestion
added: REPLACEME
```

-->

> 

Reviewer 2: This is missing the return type. Could you include it, even though the text says that an array is returned.

> Returns an array of all currently active timers by duration

Reviewer 2: ```suggestion
Returns an array of all active timers sorted by duration
```

> from shortest to longest. Order of timers with the same

Reviewer 2: ```suggestion
from shortest to longest. Timers with the same duration are sorted by ascending expiration time.
```

> duration will be based on expiration time in ascending order.

Reviewer 2: An example would be good to have.
